CONTRIBUTING.md

```diff
@@ -191,6 +191,16 @@ Creating a new release includes the following steps:
 11. The release will be automatically published to [PyPI](https://pypi.org/project/pycytominer/) via Github Actions.
 12. Manually create the release at [conda-forge](https://anaconda.org/conda-forge/pycytominer).
 
+#### Docker Hub Image Releases
```

Comment:
Dropping by with a quick request to make sure to use active voice (instead of passive voice) when possible.

Reply:
Thanks! Will get this updated with active voice in mind.